src/settings.h

```diff
@@ -202,9 +202,10 @@
 	//#################### Settings for optional Modules##############################
 	// (optinal) Neopixel
 	#ifdef NEOPIXEL_ENABLE
-		#define NUM_LEDS			24          	// number of LEDs
-		#define CHIPSET				WS2812B     	// type of Neopixel
-		#define COLOR_ORDER			GRB
+		#define NUM_INDICATOR_LEDS		24          	// number of LEDs
+		#define NUM_STATIC_LEDS			0				// not used currently
```

Comment:
What's the purpose of `NUM_STATIC_LEDS`?

Reply:
A remainder from my test codes to check, that I do not change anything expect the CRGBSet. 
I left the define in to show, how additional LEDs can be added to the LED array, f.e. #200.